Dockerfile

```diff
@@ -1,9 +1,19 @@
+FROM amd64/ubuntu:xenial@sha256:a3785f78ab8547ae2710c89e627783cfa7ee7824d3468cae6835c9f4eae23ff7
+RUN echo "disabled"
+
+FROM arm64v8/ubuntu:xenial@sha256:70fa660340a344b46cc56b3606dc8abd3bf48b5cbce13d01c720e9793a6bc3c0
+RUN echo "disabled"
+
+FROM arm32v7/ubuntu:xenial@sha256:b722e2654241f9681f4719dce7aa16a2f0c35769e17a636f5b39a33967d1aeb8
```

Comment:
@hutchic can you write to me in finger words here (again) why we're stuck at xenial (if intentional)?

Reply:
we _might_ not be explicitely stuck at xenial it just needs to be a deb based docker in general to cover rpm / deb options. For reasons it's kind of challenging to use rpm distributions to make deb but not vice versa

Reply:
Hmm okay. I guess we can address an upgrade beyond `xenial` here in later work.